org/diceresearch/nebula/orchestrator.py

```diff
@@ -2,140 +2,141 @@
 import logging
 import threading
 from datetime import datetime
-from json import JSONDecodeError
 
-import claimworthinesschecker
-import claimworthinesscheckerdummy
-import databasemanager
-import evidenceretrieval
+from claim_worthiness_check import dummy_claim_check, claim_buster
+from coref_resolution import coreference_resolution
+from database import databasemanager
+from evidence_retrieval import elastic_search
 import settings
-import stancedetection
-import translator
+from stance_detection import cosine_similarity
+from translation import neamt_translator
+from indicators.main import run_indicator_check_text
+from utils.util import SetEncoder
 from veracity_detection import predictions
 
 from utils import notification
 
 
 def goNextLevel(identifier):
-    logging.info("Orch:" + identifier)
+    """
+    Increments and executes next stage
+    :param identifier: ID
+    :return:
+    """
     current = databasemanager.getOne(settings.results_table_name, identifier)
-    STAGE_NUMBER = current[1]
-    INPUT_TEXT = current[2]
-    INPUT_LANG = current[3]
-    TRANSLATED_TEXT = current[4]
-    CLAIM_CHECK_WORTHINESS_RESULT = current[6]
-    EVIDENCE_RETRIEVAL_RESULT = current[8]
-    STANCE_DETECTION_RESULT = current[10]
-
-    #notification
-    REGISTRATION_TOKEN = current[14]
+
+    stage_number = current[1]
```

Comment:
Using these IDs looks a little bit error prone to me... I am not sure whether we want to stick to it or whether we should look for an alternative :thinking: 

Maybe that is something that we want to discuss :slightly_smiling_face:

Reply:
That is true, maybe I can change the SELECT query to return a json object instead? It would save us from having to change these indices every time we modify the database.

Reply:
I've modified it to return a dictionary. So we can access them by column name now: 
`stage_number = current["STAGE_NUMBER"]`